dist/tools/sliptty/start_network.sh

```diff
@@ -119,7 +119,7 @@ while getopts ehI: opt; do
 done
 
 PREFIX="$1"
-shift
+shift $(( $# > 0 ? 1 : 0 ))
```

Comment:
Why not just move the `if` below above this shift? Far more readable and saving some logic indirections ;-).